roles/redhat-subscriptions/tasks/activation_key.yml

```diff
@@ -0,0 +1,14 @@
+---
+- name: Install katello-ca-consumer-latest
+  when:
+    - ansible_distribution == "RedHat"
+    - ansible_distribution_major_version == "7"
+  yum:
+    name: "{{ dogfood_url }}/pub/katello-ca-consumer-latest.noarch.rpm"
+    state: present
+
+- name: Register with activationkey and consume subscriptions from Dogfood Server.
+  redhat_subscription:
+    state: present
+    activationkey: "{{ dogfood_ak }}"
```

Comment:
Minor suggestion to rename this to `dogfood_akey`. I would also argue for not tieing this to dogfood and follow best practices. That is, change this to be `redhat_subscriptions_activation_key` and `redhat_subscriptions_organization`. That allows the role to be more generic.